_modules/deployment.py

```diff
@@ -92,6 +94,9 @@ def deploy(name,
         'deploy_cmd': None,
         'test_cmd': None,
         'ok': False,
+        'version_number': version_number,
+        'build_date': generate_tag(),
```

Comment:
And to be picky: Can we rename this function to make it clearer what this does? We are using the fact it's a date as a side effect. Either we should rename `generate_tag` or put this an strftime/date op directly here.